src/main/resources/portal.properties.EXAMPLE

```diff
@@ -292,7 +292,7 @@ oncoprint.defaultview=patient
 oncoprint.oncokb.default=true
 oncoprint.hotspots.default=true
 # oncoprint.hide_vus.default=true
-# oncoprint.clinical_tracks.show_by_default=classpath:/oncoprint-default-tracks.json
+# oncoprint.clinical_tracks.config_json=classpath:/oncoprint-default-tracks.json
```

Comment:
We can merge this but we should think about whether we want this to be separate json file or perhaps part of a larger json configuration file.  @inodb don't we already have that?